File: Java/prog_lab_7/src/prog_lab_7/angle.java
package prog_lab_7;

public class angle {
	point ftpoint, verpoint, sdpoint;
	
	public angle() {
		ftpoint = new point();
		verpoint = new point();
		sdpoint = new point();
	}
	
	public void vvod() {
		this.ftpoint.vvod();
		this.verpoint.vvod();
		this.sdpoint.vvod();
	}
	
	public void vivod()	{
		System.out.printf("(%f; %f;   %f; %f;   %f; %f)\n", this.ftpoint.getx(), this.ftpoint.gety(), this.verpoint.getx(), this.verpoint.gety(), this.sdpoint.getx(), this.sdpoint.gety());
	}

	public double value() {
		return Math.acos(((this.ftpoint.getx() - this.verpoint.getx()) * (this.sdpoint.getx() - this.verpoint.getx()) + (this.ftpoint.gety() - this.verpoint.gety()) * (this.sdpoint.gety() - this.verpoint.gety())) / this.ftpoint.distance(verpoint) / this.sdpoint.distance(verpoint)) / Math.PI * 180;
	}
}

File: C#/prog_lab_6/angle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace prog_lab_6
{
    internal class angle
    {
        private point ftpoint, verpoint, sdpoint;
        public angle()
        {
            ftpoint = new point();
            verpoint = new point();
            sdpoint = new point();
        }
        public void vvod()
        {
            ftpoint.vvod();
            verpoint.vvod();
            sdpoint.vvod();
        }
        public void vivod()
        {
            Console.Write($"Первая точка угла: ");
            ftpoint.vivod();
            Console.Write($" Вершина точка угла: ");
            verpoint.vivod();
            Console.Write($" Вторая точка угла: ");
            sdpoint.vivod();
            Console.WriteLine();
        }
        public double value()
        {
            return Math.Acos(((ftpoint.getx() - verpoint.getx()) * (sdpoint.getx() - verpoint.getx()) + (ftpoint.gety() - verpoint.gety()) * (sdpoint.gety() - verpoint.gety())) / ftpoint.distance(verpoint) / sdpoint.distance(verpoint)) / Math.PI * 180;
        }
    }
}